webserver/webserver/pom.xml

```diff
@@ -161,5 +161,11 @@
             <artifactId>helidon-config-testing</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>${version.lib.okhttp3}</version>
```

Comment:
Why do you have to specify the version, isn't that managed by the helidon-project parent pom ?

Reply:
Will drop version.